sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/FilterPushdownBenchmark.scala

```diff
@@ -104,6 +107,10 @@ object FilterPushdownBenchmark extends BenchmarkBase with SQLHelper {
     df.write.mode("overwrite")
       .option("parquet.block.size", blockSize).parquet(parquetPath)
     spark.read.parquet(parquetPath).createOrReplaceTempView("parquetTable")
+
+    df.write.mode("overwrite").save(inMemoryTablePath)
```

Comment:
Cache `inMemoryTable` from file to avoid the performance issue: https://github.com/apache/spark/pull/23027#pullrequestreview-175054485